test/passing/module_attributes.ml.ref

```diff
@@ -5,14 +5,11 @@ include struct
 end
 [@warning "item"] [@@warning "structure"]
 
-include M
-[@warning "item"] [@@warning "structure"]
+include M [@warning "item"] [@@warning "structure"]
 
-include (M : S)
-[@warning "item"] [@@warning "structure"]
+include (M : S) [@warning "item"] [@@warning "structure"]
 
-include M (N)
-[@warning "item"] [@@warning "structure"]
+include M (N) [@warning "item"] [@@warning "structure"]
```

Comment:
Is this expected ? The diff doesn't look bad but I wonder why it's related to the issue.

Reply:
Yes it is, before the structure_item would not always be put into a hvbox, now it is, and it is consistent with the following lines in this test!

Reply:
Thanks